### PR TITLE
Scc 3952/add ptype deliverableto

### PR DESCRIPTION
**Pull request by charmingduchess**
- Add scholarRoom property to ptype object
- update fixtures to use nypl core data that includes scholar room
- fix typo that hasn't been a problem yet in _getFulfillmentEntities

### CODE REVIEW

**Comment by nonword** [comment]
Seems the conflicts are just version number confusion

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-6
+v14
```
Comment (nonword) [inline .nvmrc:1]: We don't want to change the Node version for `v1-node6`

File: package.json [hunk 2]
```diff
@@ -1,17 +1,19 @@
 {
   "name": "@nypl/nypl-core-objects",
-  "version": "1.6.1",
+  "version": "2.3.1",
```
Comment (nonword) [inline package.json:3]: This isn't the right version for`v1-node6`

**Review by nonword — CHANGES_REQUESTED** [review]
I think the README guidance isn't complete about how to create two parallel PRs for the two target branches (`master`, and `v1-node6`). You might want to create a `SCC-3952/add-ptype-deliverableto-v1-node6` branch from `SCC-3952/add-ptype-deliverableto` that sets the `.nvmrc` and package version correctly (and re-runs `nvm use; npm i` locally to ensure `package-lock` refers to v6 compatabible libs)

**Comment by charmingduchess** [comment]
i have the new branch i just havent closed this and pushed up yet!

**Pull request closed without merging**